src/api/max/maxServeService.ts

```diff
@@ -83,6 +83,12 @@ class MaxServeService {
       cleanup: false,
       forceKillAfterDelay: false,
       all: true,
+      env: {
+        MAX_SERVE_LOGS_OTLP_LEVEL: "DEBUG",
+        MAX_SERVE_DEPLOYMENT_ID: `INTERNAL_MODULAR_CONSOLE`,
+        MAX_SERVE_METRIC_LEVEL: "DETAILED",
+        MAX_SERVE_METRIC_RECORDING_METHOD: "PROCESS",
```

Comment:
if you want to send metrics to motel, you need to add a deployment id `MAX_SERVE_DEPLOYMENT_ID` & user id `MODULAR_USER_ID`